tools/set-env-var.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs')
+const os = require('os')
+const path = require('path')
+
+const setEnvVariable = (key, value) => {
+  const ENV_VARIABLES = fs.readFileSync(path.resolve(__dirname, '../.env'), 'utf-8').split(os.EOL)
+  const target = ENV_VARIABLES.indexOf(ENV_VARIABLES.find(line => line.match(new RegExp(`^${key}`))))
```

Comment:
Using the regex `^${key}=` would better future proof this code. With the current regex, `foo` would also match `foobar`, which is wrong, and leads to the wrong result depending on the order of the keys in the file.